gh2/csv.py

```diff
@@ -206,25 +214,23 @@ def set_headers(repo, labels=None):
         headers.extend('Label: ' + label.name for label in labels)
     return headers
 
+def get_all_labels(repositories, token):
+    labels = []
+    for repository in repositories:
+        repo_owner, repo_name = repository.split('/', 1)
+        repo = get_repo(repo_owner, repo_name, token)
```

Comment:
As mentioned elsewhere, I think it would be better if repositories was a list of the repo objects. This would save having to pass the token into this function and call get_repo here.